RealTimeFaceMaskDetector/LogViewer/Filter.h

```diff
@@ -0,0 +1,30 @@
+#ifndef LOGVIEWER_FILTER_H
+#define LOGVIEWER_FILTER_H
+#include <string>
+#include "structures.h"
+
+namespace logger {
+	class Filter {
```

Comment:
Add please description of what this class do)

Reply:
Already)